invertedai/large/common.py

```diff
@@ -85,3 +96,14 @@ def is_inside(
         x, y = point.x, point.y
         region_x, region_y = self.center.x, self.center.y
         return region_x - self.size/2 <= x and x <= region_x + self.size/2 and region_y - self.size/2 <= y and y <= region_y + self.size/2
+
+    def convert_attributes_to_properties(self,attributes):
```

Comment:
It seems this method could be called in a few other places as well. Consider moving it to utils and call it everywhere you need to convert attributes into properties.

Reply:
Done.